emulator/start_test.sh

```diff
@@ -1,6 +1,8 @@
+#!/bin/bash
+
 mkfifo /tmp/fff
 INPUT=exasol_emulator/input.csv
 SCRIPT=exasol_emulator/script.py
 LANGUAGE=python
 python exasol_emulator/exasolution.py unix:/tmp/fff $INPUT exasol_emulator/output.csv $SCRIPT &
-$1/exaudfclient ipc:///tmp/fff lang=$LANGUAGE
+"$1/exaudfclient" ipc:///tmp/fff lang=$LANGUAGE
```

Comment:
Reason:
^-- SC2086: Double quote to prevent globbing and word splitting.

Reply:
yep, before it was buggy, with path names with wildcards